azurelinuxagent/ga/exthandlers.py

```diff
@@ -441,7 +441,17 @@ def handle_ext_handler(self, ext_handler, etag):
     def handle_handle_ext_handler_error(self, ext_handler_i, e, code=-1):
         msg = ustr(e)
         ext_handler_i.set_handler_status(message=msg, code=code)
-        ext_handler_i.report_event(message=msg, is_success=False, log_event=True)
+
+        if isinstance(e, ExtensionDownloadError):
+            self.get_artifact_error_state.incr()
+            if self.get_artifact_error_state.is_triggered():
```

Comment:
ok to send the event periodically, but the error should still be logged, right?

Reply:
For `report_event` function, the default value for param `log_event` is True. Which in this case (as `is_success` is False) would log the line as well. I'll make it explicit.